cogs/reward.py

```diff
@@ -123,40 +108,29 @@ def convert_to_seconds(s):
+                print(tpString)
             
             totalGold = charDict["GP"] + treasureArray[2]
-            tpGained = treasureArray[1].values()
-            if len(tpGained) == 1:  # Case for when the character stays in the tier
-                await channel.send(content=f"A {durationString} game would give **{charDict['Name']}** \n{treasureString}\n**{charDict['Name']}** will be level {int(resultLevel)} with {int(nextCP)} CP with an additional {treasureArray[1].pop(tierInitial)} T{tierNum+1} TP and {totalGold} gold total!")
-            elif len(tpGained) == 2: # Case for crossing just one tier
-                await channel.send(content=f"A {durationString} game would give **{charDict['Name']}** \n{treasureString}\n**{charDict['Name']}** will be level {int(resultLevel)} with {int(nextCP)} CP with an additional {treasureArray[1].pop(tierInitial)} T{tierNum+1} TP, {treasureArray[1].pop(tierNext)} T{tierNum+2} TP, and {totalGold} gold total!")
-            elif len(tpGained) == 3: # Rest are cases for cheeky bastards who think they'll be in absurdly long games
-                await channel.send(content=f"A {durationString} game would give **{charDict['Name']}** \n{treasureString}\n**{charDict['Name']}** will be level {int(resultLevel)} with {int(nextCP)} CP with an additional {treasureArray[1].pop(tierInitial)} T{tierNum+1} TP, {treasureArray[1].pop(tierNext)} T{tierNum+2} TP, {treasureArray[1].pop(tierNext1)} T{tierNum+3} TP, and {totalGold} gold total! Did you remember to hydrate?")
-            elif len(tpGained) == 4:
-                await channel.send(content=f"A {durationString} game would give **{charDict['Name']}** \n{treasureString}\n**{charDict['Name']}** will be level {int(resultLevel)} with {int(nextCP)} CP with an additional {treasureArray[1].pop(tierInitial)} T{tierNum+1} TP, {treasureArray[1].pop(tierNext)} T{tierNum+2} TP, {treasureArray[1].pop(tierNext1)} T{tierNum+3} TP, {treasureArray[1].pop(tierNext2)} T{tierNum+4} TP, and {totalGold} gold total! Please, check on the kids.")
-            elif len(tpGained) == 5:
-                await channel.send(content=f"A {durationString} game would give **{charDict['Name']}** \n{treasureString}\n**{charDict['Name']}** will be level {int(resultLevel)} with {int(nextCP)} CP with an additional {treasureArray[1].pop(tierInitial)} T{tierNum+1} TP, {treasureArray[1].pop(tierNext)} T{tierNum+2} TP, {treasureArray[1].pop(tierNext1)} T{tierNum+3} TP, {treasureArray[1].pop(tierNext2)} T{tierNum+4} TP, {treasureArray[1].pop(tierNext3)} T{tierNum+5} TP, and {totalGold} gold total! Of course, you will die of exhaustion from playing that much in one game, but yay tier 5???")
+            # Final output
+            await channel.send(content=f"A {durationString} game would give **{charDict['Name']}** \n{treasureString}\n**{charDict['Name']}** will be level {int(resultLevel)} with {int(resultCP)} CP with an additional {tpString}and {totalGold} gold total!")
```

Comment:
Do you really want to give up on your comments or would like some advice on how to keep them?

Reply:
I'd love advice, since I am self-conscious about going overboard with documentation.

Reply:
Oh, I meant the snarky little comments you had for when people crossed a lot of tiers.

The documentation style that I learned in school was to document each and every line, but my first work experience basically said "the time you spent documenting could have been spent on coding" so I am not sure what to tell you.
My personal policy is that documentation for lines of code is only necessary when it is not immediately obvious what it does.

I documented timer.py so you could check out what I wrote there.

For including your fun comments you could create an array that contains the messages and index it by tier counts. That would replicate what you had earlier.

Reply:
For timer.py I went with the "document every line policy" though...